scripts/trainer.py

```diff
@@ -78,6 +84,17 @@ def parse_args(args):
     parser = argparse.ArgumentParser(description=description)
 
     parser.add_argument("model", help="Which model to train.")
+    parser.add_argument(
+        "--limit",
+        type=int,
+        help="Only train on a subset of the data, used mainly for integrations tests",
+    )
+    parser.add_argument(
+        "--no-download",
+        action="store_false",
+        dest="download_db",
+        help="Do not download databases, uses whatever is on disk",
+    )
```

Comment:
I would prefer an explicit option so it would fails more loudly, for example if we rename the BUGS_DB but have a mismatch between the bug_retriever and the trainer, with the proposition it would download the bugs DB silently.